docs/index.md

```diff
@@ -1,30 +1,15 @@
 # Welcome to Ultra
 - - -
-Ultra is an application development platform that enables external service providers such as MakeMyTrip and Phone Pe to host their application services on Flipkart. It helps Flipkart customers to securely access a variety of online convenience services such as online ticketing and payment of bills. The platform is scalable to include other business categories into Flipkart domain.
+Ultra is an application development platform that enables external service providers such as MakeMyTrip and Phone Pe (we call them partners in Ultra) to host their application services on Flipkart. It helps Flipkart customers to securely access a variety of online convenience services such as online ticketing and payment of bills. The platform is scalable to include other business categories into Flipkart domain.
 
-It involves on-boarding of the external companies or organizations (also called as Ultra partners or simply partners) who specialise in the categories like travel ticket reservations, booking online movie tickets, paying online bills and more to improve their customer base through Flipkart. Ultra redirects a user to their application within Flipkart and grants the entry based on the user’s login credentials after seeking permissions from the user to share the data further. The users and the partners carry out the transactions through Flipkart payment gateway (FKPG). Post the successful transactions between them, users can check the status of these activities in Flipkart app.
+It involves on-boarding of partners’ application who specialise in the categories like travel ticket reservations, booking online movie tickets, paying online bills, etc. to improve their customer base through Flipkart. Ultra redirects a user to a partner’s application within Flipkart app window. The entry in the partner’s app is allowed based upon the user’s login credentials and permissions for sharing their data further. The users and the partner carry out all transactions through the Flipkart payment gateway (FKPG). After successful transactions, the user is able to check the status of the placed orders in Flipkart.
 
-Using Ultra as a development platform allows such external companies to open their doors to over 100 million handhelds in the country. Ultra supports a wide variety of technologies from the most basic web applications to very advanced technologies powered by React Native, thus, provides ample room to grow and develop unique skills while keeping entry barrier low such that it’s not overwhelming.
```

Comment:
React native is a technology, not a technique

Reply:
Changed techniques to technologies. Please check.